packages/core/src/hydration/event_replay.ts

```diff
@@ -79,3 +118,78 @@ export function setJSActionAttribute(
     }
   }
 }
+
+function setEventReplayer(dispatcher: Dispatcher) {
+  dispatcher.setEventReplayer(queue => {
+    for (const event of queue) {
+      handleEvent(event);
+    }
+  });
+}
+
+/**
+ * TODO: Update documentation here.
+ *
+ * @param target Component, Directive or DOM Node.
+ */
+export function getLViewByElement(target: any): LView|null {
+  let lView = readLViewData(target);
```

Comment:
nit: I'd propose dropping `Data` from here:
```suggestion
  let lView = readLView(target);
```

Reply:
done